slices/perl-base.yaml

```diff
@@ -0,0 +1,32 @@
+package: perl-base
+
+essential:
+  - perl-base_copyright
+
+slices:
+  modules:
+    contents:
+      /usr/lib/*-linux-*/perl-base/**/*.pl:
+      /usr/lib/*-linux-*/perl-base/**/*.pm:
+      /usr/lib/*-linux-*/perl-base/*.pl:
+      /usr/lib/*-linux-*/perl-base/*.pm:
+
+  libs:
+    essential:
+      - libc6_libs
+      - libcrypt1_libs
+    contents:
+      /usr/lib/*-linux-*/perl-base/**/*.so:
+
+  bins:
+    essential:
+      - perl-base_libs
+      - perl-base_modules
+    contents:
+      # hard link
+      /usr/bin/perl:
+      /usr/bin/perl5.38.2: {copy: /usr/bin/perl}
```

Comment:
I think we are copying the hard links as a workaround before chisel officially supports hard links. An example would be https://github.com/canonical/chisel-releases/blob/ubuntu-22.04/slices/python3-lib2to3.yaml

Reply:
Since this is just the bin directory, and the original file probably wont be deleted, could we just use symlinks instead? similar to how python and go are linked.

Reply:
`/usr/bin/perl` is a regular file in the package tarball, while the `/usr/bin/perl5.38.2` is a hardlink to `/usr/bin/perl` in that tarball. To make sure that any modifications (e.g., remove or re-link) to `/usr/bin/perl` won't invalidate `/usr/bin/perl5.38.2`, we have to copy it. Further optimizations and avoiding copying rely on chisel's ability to handle hardlinks.